src/ensembl/src/content/app/species/components/species-stats/SpeciesStats.scss

```diff
@@ -0,0 +1,45 @@
+@import 'src/styles/common';
+
+$secondary-text-color: $dark-grey;
+
+.wrapper {
+  width: 250px;
```

Comment:
Wondering whether there is any benefit to setting the width and the display: inline-block on these elements, or whether the grid of the parent element will provide those. I would probably just go with the grid of the parent. But can't think of a convincing argument one way or the other.

Reply:
As discussed during the sprint review, we will look into this when we start using this component and change it to control the the width using grid of the parent if necessary.